common/remote/sockfile.cpp

```diff
@@ -4576,22 +5031,52 @@ void verifyMetaInfo(IPropertyTree &actNode, bool authorizedOnly, const IProperty
 #endif
         metaInfo.set(metaInfoEnvelope);
 
-    IPropertyTree *fileInfo = metaInfo->queryPropTree("FileInfo");
-    assertex(fileInfo);
-
-    // extra filename based on part/copy.
     assertex(actNode.hasProp("filePart"));
     unsigned partNum = actNode.getPropInt("filePart");
```

Comment:
This code seems to be executed for both read & write. However, the write JSON format template doesn't have filePart as a requirement. Do we need to specify filePart for write?

Reply:
Yes, needs to be present whether reading or writing.
I'll update the examples + add some notes.